.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
The interval controls how often the project's dependencies are scanned. So every day it should look for new releases in `github.com/elastic/*` deps, and if it hasn't already raised a PR then it will open one for that dependency. It should not open more than one update PR per dependency version.

Having a longer interval would negate some of the utility for having this automation. If you fix a bug specifically for Beats in an upstream dependency then you probably don't want to wait up to one week for that change to be incorporated. Personally I would probably do the update manually before waiting that long. If there were a way to manually trigger dependabot then that could be a workaround.

If you are concerned about noise then we could set `open-pull-requests-limit: 2`. Then the bot would have at most two open PRs. We would need to merge or close some PRs before it will open more. WDYT?